src/Promitor.Agents.Scraper/Validation/Factories/MetricValidatorFactory.cs

```diff
@@ -33,6 +33,8 @@ internal static IMetricValidator GetValidatorFor(ResourceType resourceType)
                     return new CosmosDbMetricValidator();
                 case ResourceType.DataFactory:
                     return new DataFactoryMetricValidator();
+                case ResourceType.DataExplorerCluster:
```

Comment:
Nit - Can you alphabetically sort please?

Reply:
Fixed nit